templates/emails/user.monthlyreport.hbs

```diff
@@ -238,6 +238,13 @@ Subject: {{{capitalize month}}} Report
               </li>
             {{/if}}
           {{/each}}
+          {{#if tags.opensource}}
+            {{#if ORGANIZATION}}
+              <li><font color="red"><b>NEW!</b></font> Check your profile on <a href="https://backyourstack.com/{{fromCollective.slug}}">backyourstack.com/{{fromCollective.slug}}</a> to find out what are the open source projects that your organization is depending on and that need financial support! (note: you may need to adapt the url to match your organization slug on Github)</li>
+            {{else}}
```

Comment:
It should not be `https://backyourstack.com/{{fromCollective.slug}}` but something like https://backyourstack.com/{{fromCollective.settings.githubOrg}}

Reply:
@xdamman have you seen that comment?